src/main/api/exchangeOAuthCodeForAccessTokenUsingPKCE.json

```diff
@@ -0,0 +1,64 @@
+{
+  "uri": "/oauth2/token",
+  "comments": [
+    "Exchanges an OAuth authorization code for an access token.",
+    "If you will be using the Authorization Code grant, you will make a request to the Token endpoint to exchange the authorization code returned from the Authorize endpoint for an access token."
```

Comment:
Should we update this to indicate this version of the API takes the `code_verifier` for PKCE?

Reply:
Yup, good catch.